mozilla-release/mobile/android/base/java/org/mozilla/gecko/BrowserApp.java

```diff
@@ -4821,5 +4831,23 @@ private void refreshDrawables(){
     public boolean setRequestedOrientationForCurrentActivity(int requestedActivityInfoOrientation) {
         return super.setRequestedOrientationForCurrentActivity(requestedActivityInfoOrientation);
     }
+
+    private void setLightTheme() {
```

Comment:
You are setting nothing, the name is misleading. A better one will be `updateTheme`.